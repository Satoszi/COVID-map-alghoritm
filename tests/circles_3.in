14 14 2 63
circles_3
+------------+
|+----------+|
||+--------+||
|||+------+|||
||||+----+||||
|||||+--+|||||
||||||++||||||
||||||++||||||
|||||+--+|||||
||||+----+||||
|||+------+|||
||+--------+||
|+----------+|
+------------+