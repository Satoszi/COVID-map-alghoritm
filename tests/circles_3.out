11 13
8 13
5 13
2 13
8 12
5 12
2 12
0 12
13 11
12 11
11 11
8 11
5 11
2 11
8 10
5 10
1 10
5 9
3 9
0 9
13 8
12 8
11 8
10 8
9 8
8 8
5 8
2 8
4 7
1 7
6 6
3 6
0 6
13 5
12 5
11 5
10 5
9 5
8 5
5 5
2 5
7 4
4 4
1 4
9 3
6 3
3 3
0 3
13 2
12 2
11 2
8 2
5 2
2 2
10 1
7 1
4 1
1 1
12 0
9 0
6 0
3 0
0 0
